src/shared/sol-util.c

```diff
@@ -905,3 +905,79 @@ sol_util_uuid_str_valid(const char *str)
 
     return true;
 }
+
+SOL_API int
+sol_util_escape_quotes(const struct sol_str_slice slice,
```

Comment:
I will keep as int, because in the case that you described I will transform x"y" into xy.